### PR TITLE
Make opensource ready

### PR DESCRIPTION
**Pull request by frederikja163**

### CODE REVIEW

**Comment by frederikja163** [comment]
#22 is blocking this PR somewhat, this PR adds documentation for things in that PR. Namely the path parameter of the result listener. Also closes #23

File: .github/workflows/pr-comment.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+jobs:
+  pr-version-comment:
+    runs-on: ubuntu-latest
+    name: pr-version-comment
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # This action needs the entire history of the repository to calculate the version
+          fetch-depth: 0
```
Comment (db-ks) [inline .github/workflows/pr-comment.yml:23]: We will see if this works, when we tag the release.

**Comment by db-ks** [comment]
Is the default filename templated to use the `ResultType`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Write `tap package install Parquet` to install the newest release of the Parquet plugin.
+If you are using OpenTAP in a runner it might already be installed if you are using the latest version.
+
+## Writing results
+Add the result listener called `Parquet` to your test plan.
+These are the options available on the result listener at this moment.
+
+| Name | Description | Default value |
+|-|-|-|
+| File path  | The file path of the parquet file(s). Can use \<ResultType> to have one file per result type. | Results/\<TestPlanName>.\<Date>.parquet |
```
Comment (db-ks) [inline README.md:14]: Can you list other options or link to the OpenTAP documentation that describes more details about the possible templates?

**Comment by frederikja163** [comment]
> Is the default filename templated to use the `ResultType`?

No it is not, it uses the testplan name and the datetime of the testplan.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Depending on what you have specified within the file path, a single test plan could result in one or more parquet files. All steps will allways generate atleast one row within a parquet file, and the test plan will always create one and only one row.
```
Comment (db-ks) [inline README.md:16]: atleast -> at least

**Review by db-ks — CHANGES_REQUESTED** [review]
Few comments and couple of suggestions.

**Review by db-ks — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Write `tap package install Parquet` to install the newest release of the Parquet plugin.
+If you are using OpenTAP in a runner it might already be installed if you are using the latest version.
+
+## Writing results
+Add the result listener called `Parquet` to your test plan.
+These are the options available on the result listener at this moment.
+
+| Name | Description | Default value |
+|-|-|-|
+| File path*  | The file path of the parquet file(s). Can use \<ResultType> to have one file per result type. Further documentation can be found here: https://doc.opentap.io/Developer%20Guide/Appendix%20A/#result-listeners | Results/\<TestPlanName>.\<Date>.parquet |
```
Comment (frederikja163) [inline README.md:14]:
```suggestion
| File path*  | The file path of the parquet file(s). Can use \<ResultType> to have one file per result type. Further documentation can be found [here](https://doc.opentap.io/Developer%20Guide/Appendix%20A/#result-listeners) | Results/\<TestPlanName>.\<Date>.parquet |
```

**Pull request merged**